### ISSUE TITLE
Skillet not working window with Leatherworking in Spanish version

### ISSUE THREAD

**Issue by Zaruth** [b-morgan/Skillet-Classic#153]
The window opens with all professions except Leatherworking in the Spanish version.
As you can see here, it opens Cooking without problems.
![image](https://github.com/b-morgan/Skillet-Classic/assets/36704169/d0960ae0-f17a-48b4-b27b-97ea94792f17)

But here, you can see that it does not open the interface with Leatherworking.
![image](https://github.com/b-morgan/Skillet-Classic/assets/36704169/fab7840f-1057-4943-804f-95fc7d2e42ba)

Any ideas? Maybe it's because of the double translation that the game uses for the Leatherworking profession, and that's why it's not finding the profession.

**Comment by b-morgan** [comment]
In SkilletData.lua lines 68-83 the following table exists:
```
--
--  a table of locale specific translations by id
-- needed to fix Blizzard inconsistent translations
--
-- [tradeID] = {locale, old, new}
--   locale is what GetLocale() returns
--   old is the return from GetSpellInfo(tradeID)
--   new is the return from GetTradeSkillLine() when the tradeskill / craft is opened
--
local TranslateList = {
	[4036] = {"frFR", "Ingénieur", "Ingénierie"},			-- engineering
	[3273] = {"frFR", "Premiers soins", "Secourisme"},		-- first aid
	[2108] = {"esES", "Peletería", "Marroquinería"},		-- leatherworking
	[3908] = {"esES", "Sastrería", "Costura"},				-- tailoring
	[2108] = {"koKR", "가죽세공", "가죽 세공"},					-- leatherworking
}
```
As you can see, there is already an entry for leatherworking in the enES locale. Since it isn't working for you, Blizzard may have messed with their translations. I need from you the exact output of `/dump GetLocale()`, `/dump GetSpellInfo(2108)`, and `/dump GetTradeSkillLine()` when Blizzard's leatherworking UI is visible.

Feel free to edit the table yourself to verify that it works. Unfortunately, I can't test it myself so I will need your help.

**Comment by b-morgan** [comment]
If the line numbers are slightly different it is because the current released version for Classic Era is 1.74 (or 1.75-alpha4 which will work on both 1.14.3 and 1.14.4) and you are using 1.70. The table in question hasn't changed since 1.70.

**Comment by camador** [comment]
I'm not the post owner, but here is my game output:

![skillet](https://github.com/b-morgan/Skillet-Classic/assets/4353411/9129a8ee-d9e9-4aeb-848d-a0a886d437a4)

Maybe this can help:

![skillet_open_leatherworking](https://github.com/b-morgan/Skillet-Classic/assets/4353411/bbe18240-9344-47eb-8f76-14d6bdd9282a)

**Comment by b-morgan** [comment]
@camador, Thank you for your help!

The output you provided appears to be identical to the existing entry in the table. I have attached a modified Skillet.lua to add some additional debug statements. Please install that file (while WoW is closed), type "/skillet tabledump on", "/skillet debuglevel 2", open Leatherworking, and give me the debug output again. Please also attach your character specific saved variables file Skillet-Classic.lua which will contain the debug output with the special characters in the names.

[Skillet.zip](https://github.com/b-morgan/Skillet-Classic/files/12476666/Skillet.zip)

You can type "/skillet debugoff" and "/skillet cleardebuglog" when you are done so the debug output doesn't spam your chat.

**Comment by camador** [comment]
Sure, here they are:

![dump](https://github.com/b-morgan/Skillet-Classic/assets/4353411/a730f7bc-8567-4c3c-8736-3ffdc2d3745c)

[Skillet-Classic.zip](https://github.com/b-morgan/Skillet-Classic/files/12482245/Skillet-Classic.zip)

**Comment by b-morgan** [comment]
Please change line 82 in SkilletData.lua from:
`	[2108] = {"koKR", "가죽세공", "가죽 세공"},					-- leatherworking`
to:
`--	[2108] = {"koKR", "가죽세공", "가죽 세공"},					-- leatherworking`

I need to use a different table layout which I'll work on and release a new build.

**Comment by b-morgan** [comment]
This is fixed in [Skillet-Classic-1.77](https://legacy.curseforge.com/wow/addons/skillet-classic/files/4730889).

**Comment by camador** [comment]
Yes, that did the trick.

Thank you!
